Uyanda.Coffee.Persistence/Accessors/BeverageAccessor.cs

```diff
@@ -71,6 +71,16 @@ public async Task<IEnumerable<BeverageSizeCostModel>> AddBeverageCostAsync(IEnum
             return entities.Select(ToModel);
         }
 
+
+        public async Task<IEnumerable<BeverageSizeCostModel>> GetBeverageCostAsync()
+        {
+            var query = await localDbContext.BeverageCost.AsNoTracking()
+                        .Select(row => row).ToArrayAsync();
```

Comment:
_Select_ is used for **projection**, when you don't need to project (like now), you can leave it out.